bi_sql_editor/models/bi_sql_view.py

```diff
@@ -281,6 +282,9 @@ def unlink(self):
                     "If you want to delete them, first set them to draft."
                 )
             )
+        for sql_view in self:
+            if sql_view.cron_id:
+                sql_view.cron_id.unlink()
```

Comment:
Nitpicking : The 3 lines could be replaced by seld.mapped("cron_id").unlink() i think.

Reply:
no problem, I also think it looks cleaner like this 👍🏿